.github/workflows/tests_lkcpu_python.yml

```diff
@@ -148,15 +148,15 @@ jobs:
         with:
           name: ${{ matrix.pl_backend }}-${{ matrix.exec_model }}_name.txt
           retention-days: 1
-          if-no-files-found: error
+          if-no-files-found: warn
```

Comment:
We'd like to get errors in case of failures instead of warning.

Reply:
Yes, but for now we just wanted to see whether the tests were passing (because all jobs get cancelled as soon as one of them fails).

Reply:
So this is not going to merge into master.

Reply:
No, or we can choose not to.